src/DotNetProjectFile.Analyzers/DotNetProjectFile.Analyzers.csproj

```diff
@@ -202,6 +204,10 @@ v1.0.0
     <None Include="README.md" Pack="true" PackagePath="" />
   </ItemGroup>
 
+  <ItemGroup Label="Props and targets">
+    <PackageReference Include="Microsoft.Sbom.Targets" Version="3.0.0" PrivateAssets="all" />
```

Comment:
Shouldn't this also be added to the list of packages that should have private assets

Reply:
Yes, it should.